GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt

```diff
@@ -56,6 +57,9 @@
 # Python
 !/**/*.py
 !/**/*.ipynb
+!/**/requirements.txt
+!/**/Pipenv
+!/**/Pipfile.lock
```

Comment:
@rcarroll901 just want to confirm with you that both of these should be un-ignored

Reply:
!Pipenv needs to be changed to !Pipfile. Otherwise, looks great.

Reply:
```suggestion
!/**/Pipfile
!/**/Pipfile.lock
```
Like this? Hit commit if correct